src/lib/components/PostDirectionChip.tsx

```diff
@@ -20,9 +22,24 @@ const useStyles = makeStyles(
 const PostDirectionChip: React.FC<IPostDirectionChipProps> = (props) => {
   const classes = useStyles(props);
   const { labelName } = props;
+  const history = useHistory();
+
+  const handleClick = () => {
+    const direction = Object.values(DIRECTION_PROPERTIES).find(
+      (d) => d.label === labelName,
+    );
+    if (direction?.route) {
+      history.push(`/direction/${direction.route}`);
+    }
+  };
```

Comment:
putting a router logic in the chip is not a good idea...
chip is a basic UI component that can have a click handling logic implemented on a higher lvl, so that the component that uses chip will handle the click and, therefore, the routing part.

Since this chip is used ONLY for the direction, we can have this for now, I suppose.. But, bear in mind that you will need chip in another part of the app, so that you will have to separate the routing and representation logic of it. 
It's not a big deal to do that, but please be aware that you will have to do that when necessary :)

also, if putting the router inside the chip (since it's **direction** chip only), its props may be 
```
{
  direction: IDirection;
}
``` 
instead. So you won't have to get all the directions and searchBy label.

Also, search by label is bad, since `label` fields usually are not unique and if you do the search, it's better to search by id or something.

please, pass the direction as a prop to the chip.

Reply:
I had some second thoughts about this part too, but left it as is until further discussion on daily.

> bear in mind that you will need chip in another part of the app, so that you will have to separate the routing and representation logic of it.

Good point, didn't notice that right away. I'll make a wrapper that adds interactive behavior then.

Reply:
fixed!